src/components/Pokemon.js

```diff
@@ -0,0 +1,48 @@
+import {useEffect, useState} from "react";
+import axios from "axios";
+import './Pokemon.css'
+
+function Pokemon( {pokemonName} ) {
+
+    const [pokemon, setPokemon] = useState({});
+
+  useEffect(() => {
+
+      async function fetchPokemon() {
+
+          try {
+              const response = await axios.get(`https://pokeapi.co/api/v2/pokemon/${pokemonName}`)
+             /* console.log(response.data);*/
+              setPokemon(response.data);
+          } catch (error) {
+              console.error(error)
+          }
+      }
+
+      if (pokemonName) {
+      fetchPokemon();
+      }
+  }, [pokemonName]) ;
+
+  const {name, moves, weight, abilities, sprites } = pokemon
```

Comment:
Omdat `pokemon` deze properties nog niet heeft (of na mijn comment zelfs `undefined` is), is het beter om `null` terug te geven voordat deze properties uitgelezen worden. Anders krijg je met undefined errors te maken
```suggestion
  if (!pokemon) return null;
  
  const {name, moves, weight, abilities, sprites } = pokemon
```
Bovenstaande oplossing is dus wanneer de default waarde `undefined` is